.cicd/pipeline.yml

```diff
@@ -154,6 +154,75 @@ steps:
 
   - wait
 
+  - label: ":aws: Amazon_Linux 2 - Toolchain Tests"
```

Comment:
Please move this test group before the test metrics step so we can collect test metrics.